service/controller/persistent_volume.go

```diff
@@ -83,9 +76,9 @@ func NewPersistentVolume(config PersistentVolumeConfig) (*PersistentVolume, erro
 			ResourceSets: []*controller.ResourceSet{
 				v1ResourceSet,
 			},
-			RESTClient: config.K8sClient.CoreV1().RESTClient(),
+			RESTClient: config.K8sClient.K8sClient().CoreV1().RESTClient(),
```

Comment:
What the difference between this and `config.K8sClient.RESTClient()`?

Reply:
With the latest `k8sclient` there might not be a problem anymore. Earlier the specific k8s client was configured with the correct scheme accordingly. Since we do this now with `k8sclient` directly we would need to test if the old way is still necessary.

Reply:
Using the configured client is a bit simpler. I'll update and test.